store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -131,20 +131,31 @@
     private Connector connection = null;
 
     @Override
-    public void initialise(final Schema schema, final StoreProperties properties) throws StoreException {
-        preInitialise(schema, properties);
+    public void initialise(final String graphId, final Schema schema, final StoreProperties properties) throws StoreException {
+        preInitialise(graphId, schema, properties);
         TableUtils.ensureTableExists(this);
     }
 
     /**
      * Performs general initialisation without creating the table.
      *
+     * @param graphId    the graph ID
      * @param schema     the gaffer Schema
      * @param properties the accumulo store properties
      * @throws StoreException the store could not be initialised.
      */
-    public void preInitialise(final Schema schema, final StoreProperties properties) throws StoreException {
-        super.initialise(schema, properties);
+    public void preInitialise(final String graphId, final Schema schema, final StoreProperties properties) throws StoreException {
+        final String deprecatedTableName = ((AccumuloProperties) properties).getTable();
+        if (null == graphId && null != deprecatedTableName) {
+            // Deprecated
+            super.initialise(deprecatedTableName, schema, properties);
```

Comment:
This will avoid making a breaking change and give users some time to make changes to their code whilst the table name property is deprecated. When we remove the table name altogether in a later version we will remove this too.